### PR TITLE
Upgrade and Fix: React Native live camera filters (0.71.7)

### PR DESCRIPTION
**Pull request by saad-abid-crowdbotics**
## Type of PR

- [ ] Bugfix
- [ ] New feature
- [x] Minor changes

## Changes introduced
Update packages for new scaffold 0.71.7 fix

### CODE REVIEW

**Comment by saad-abid-crowdbotics** [comment]
Looks good to me.

**Review by cbshoaib — DISMISSED** [review]
LGTM

**Review by danielsousaio — APPROVED** [review]

**Review by danielsousaio — COMMENTED** [review]
Needs updates to the package.json

File: modules/react-native-live-camera-filters/package.json [hunk 1]
```diff
@@ -5,15 +5,15 @@
...
+    "react-native-filter-camera": "github:saad-abid-crowdbotics/react-native-filter-camera"
   },
   "author": "Crowdbotics",
   "license": "ISC",
   "dependencies": {
-    "@react-native-community/cameraroll": "^4.1.2",
+    "@react-native-camera-roll/camera-roll": "^5.7.2",
     "react-native-vector-icons": "^9.2.0",
-    "react-native-filter-camera": "0.1.18"
+    "react-native-filter-camera": "github:saad-abid-crowdbotics/react-native-filter-camera"
```
Comment (danielsousaio) [inline modules/react-native-live-camera-filters/package.json:17]: Why is this needed?

**Review by danielsousaio — CHANGES_REQUESTED** [review]

File: modules/react-native-live-camera-filters/package.json (hunk 1 above)
Comment (shahraizali) [reply modules/react-native-live-camera-filters/package.json:17]: @saad-abid-crowdbotics can comment on this but this was related to new scaffold 0.71 support there was some issue so he had to sort of monkeypatch this

Comment (saad-abid-crowdbotics) [reply modules/react-native-live-camera-filters/package.json:17]: This module uses the "react-native-filter-camera" library to apply filters. The "react-native-filter-camera" library relies on another native android library called "com.otaliastudios:cameraview" v2.6.2 for Android. However, this version of the library is not compatible with the New Scaffold version 0.71.7. It requires v2.7.2. To address this issue, There were two solutions to this either I forked the library or change the version in "node_modules/react-native-filter-camera/android/build.gradle"

**Review by danielsousaio — APPROVED** [review]
